test/core/gprpp/time_util_test.cc

```diff
@@ -0,0 +1,135 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include <gtest/gtest.h>
+
+#include <grpc/support/time.h>
+
+#include "absl/time/time.h"
+
+#include "src/core/lib/gprpp/time_util.h"
+
+TEST(TimeUtilTest, ToGprTimeSpecFromAbslDuration) {
+  EXPECT_EQ(0,
```

Comment:
Please put each of these checks in its own test.

Reply:
Done.